patch.yaml

```diff
@@ -30,6 +44,77 @@ pipeline:
     with:
       uri: https://ftp.gnu.org/gnu/patch/patch-${{package.version}}.tar.gz
       expected-sha256: 8cf86e00ad3aaa6d26aca30640e86b0e3e1f395ed99f189b06d4c9f74bc58a4e
+
+  - uses: fetch
```

Comment:
Please copy the patches into our repository, instead of fetching them from Alpine's cgit instance.

Reply:
thanks - done